addons/website/static/src/js/utils.js

```diff
@@ -239,6 +239,27 @@ function websiteDomain(self) {
     return ['|', ['website_id', '=', false], ['website_id', '=', websiteID]];
 }
 
+/**
+ * Checks if the 2 given URLs originates from the same domain.
```

Comment:
Still don't like that assessment but let's not keep nitpicking about this :grimacing:

Reply:
I adapted the whole docstring, it should be clear now